### DISCUSSION TITLE
Entry page

### DISCUSSION THREAD

**Discussion by Corobori** [requarks/wiki#6599] [Help / Questions] [answered]
Hi,
I am brand new to wiki.
I have started a new Wiki and I would like to do some customization in my entry page. Below the introduction text, I would like to have a list of the recently added page, and in the right column, all tag's used as a link and a list of the top publisher.  Is it possible to do that ?

**Comment by Corobori** [comment]
Reading some other topics I found out the answer to my questions, at least I think so,one seems to be part of things being planned, see [here](https://feedback.js.wiki/wiki/p/sub-pages-list).

**Comment by NGPixel** [comment, marked as answer]
Not at the moment but planned for v3.
